nut/DOCS.md

```diff
@@ -35,7 +35,7 @@ comparison to installing any other Home Assistant add-on.
 1. Configure the `users` and `devices` options.
 1. Start the "Network UPS Tools" add-on.
 1. Check the logs of the "Network UPS Tools" add-on to see if everything went well.
-1. Configure the [NUT Integration][nut-ha-docs].
+1. Configure the [NUT Integration][nut-ha-docs] and use `a0d7b954-nut` hostname.
```

Comment:
This could be something else... for example, in my case, it is `40817795-nut`. But I guess this is the common one, so that is fine.

The sentence doesn't flow though add `the` before `hostname`? Also, I think HA asks for "Host" and not "hostname"?

Reply:
AFAIK the id is SHA1(addon_url)[:8], so it will be `a0d7b954-nut` for most people as long as you don't change its name or move it. But it's a good point, I rewrote it to allow for other hostnames, hopefully it flows better now.

As for the `host` or `hostname` question, the only place where `hostname` is not used is the title of the field in the integration configuration. The add-on page shows `hostname` and even the tooltip in the integration configuration says: `The hostname or IP address of your NUT server.` So I guess it's pick-your-own, the consistency is already gone and I tried now to match the field name in the right context. That being said, I can stick to using just one of them if you prefer that.